src/client/app/components/maps/MapViewComponent.tsx

```diff
@@ -127,9 +128,18 @@ class MapViewComponent extends React.Component<MapViewPropsWithIntl, MapViewStat
 
 		let toggleButton;
 		if (hasToken()) {
+			// throw out alert if the admin wants to display uncalibrated map
+			if (!(this.props.map.origin && this.props.map.opposite)) {
+				toggleButton = <Button style={this.styleToggleBtn()} color='primary' onClick={this.notifyCalibrationNeeded}>
+				<FormattedMessage id={buttonMessageId} />
+				</Button>;
+			}
+			// if map is already calibrated, the button will allow it to be displayed
+			else {
```

Comment:
When I format in Visual Studio it changes some of the indenting such as making the else statements indented. If you can to it great. If not, I could change the formatting.